man/systemd.unit.xml

```diff
@@ -979,6 +979,16 @@
         ones.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>SurviveSoftReboot=</varname></term>
+
+        <listitem><para>Takes a boolean argument. Defaults to <option>no</option>. If <option>yes</option>,
+        the unit will be configured to survive a soft reboot operation (see:
+        <citerefentry><refentrytitle>systemd-soft-reboot.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>).
+        This has the same effect as <varname>IgnoreOnIsolate=yes</varname>, and will also ensure the unit is
```

Comment:
I don't think this should imply IgnoreOnIsolate=yes. At least we shouldn't document this here.

Reply:
It is needed for the unit to survive - I can avoid mentioning the option, and instead say that it will survive 'isolate' operations or so?